app/javascript/flavours/glitch/features/local_settings/page/index.js

```diff
@@ -92,6 +94,16 @@ class LocalSettingsPage extends React.PureComponent {
         >
           <FormattedMessage id='settings.rewrite_mentions' defaultMessage='Rewrite mentions in displayed statuses' />
         </LocalSettingsPageItem>
+        <LocalSettingsPageItem
+          settings={settings}
+          item={['num_visible_reactions']}
+          id='mastodon-settings--num_visible_reactions'
+          onChange={onChange}
+          placeholder={intl.formatMessage(messages.enter_amount_prompt)}
+          number
+        >
+          <FormattedMessage id='settings.num_visible_reactions' defaultMessage='Number of visible reaction badges:' />
+        </LocalSettingsPageItem>
```

Comment:
Not sure about making this a glitch flavour only setting or feature in general.
While I don't see upstream implementing it anytime soon, I think it would be worth it to have reactions in the vanilla flavour. (Which would require this being a preference instead of local setting)

Reply:
Fair point, I'll move it.  Having reactions in the vanilla flavour also requires copying all changes in `/app/javascript/flavours/glitch` over to `/app/javascript/mastodon`, right?